src/toolchain.rs

```diff
@@ -245,11 +245,15 @@ impl<'a> Toolchain<'a> {
 
                 // some toolchains are faulty with some combinations of platforms and
                 // may fail to launch but also to timely terminate.
-                // (known cases include Rust 1.3.0 through 1.10.0 in recent macOS Sierra.)
+                // (
+                //   known cases include Rust versions
+                //     - 1.3.0 through 1.10.0 in recent macOS Sierra.
+                //     - 1.10.0 through 1.51.0 in macOS Big Surf.
+                // )
                 // we guard against such cases by enforcing a reasonable timeout to read.
                 let mut line1 = None;
                 if let Ok(mut child) = cmd.spawn() {
-                    let timeout = Duration::new(10, 0);
+                    let timeout = Duration::new(24, 0);
```

Comment:
> "a reasonable timeout to read"

> 24 seconds

I believe it would be appropriate to:
- actually solve the problem
XOR
- remove the description "reasonable", which was added when this was **1** second.

Reply:
If you go with the latter, I suggest using `Duration::from_secs` to make it explicit what the current state of affairs is at a brief glance.